guides/paper-iron-layout.md

```diff
@@ -0,0 +1,324 @@
+---
+title: Responsive layouts with Paper and Iron
```

Comment:
Should this be "with Paper and Iron Elements"?
I think you could even cut "Iron Elements" from the title, because they're really supporting characters here. 

(Could also be: Responsive material design layouts.)